examples/r2i/mean_std_preprocessing.cc

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2020 RidgeRun, LLC (http://www.ridgerun.com)
+ * All Rights Reserved.
+ *
+ * The contents of this software are proprietary and confidential to RidgeRun,
+ * LLC.  No part of this program may be photocopied, reproduced or translated
+ * into another programming language without prior written consent of
+ * RidgeRun, LLC.  The user is free to modify the source code after obtaining
+ * a software license from RidgeRun.  All source code changes must be provided
+ * back to RidgeRun without any encumbrance.
+ */
+
+#include <memory>
+#include <vector>
+
+#include <r2i/ipreprocessing.h>
+#include <r2i/onnxrt/frame.h>
+
+#define STB_IMAGE_IMPLEMENTATION
+#include "stb_image.h"
+
+#define STB_IMAGE_RESIZE_IMPLEMENTATION
+#include "stb_image_resize.h"
+
+/* Model specific required dimensions */
+#define REQ_WIDTH_224 224
+#define REQ_HEIGTH_224 224
+
+class Example: public r2i::IPreprocessing {
+ public:
+  r2i::RuntimeError Apply(std::shared_ptr<r2i::IFrame> in_frame,
+                          std::shared_ptr<r2i::IFrame> out_frame, int required_width, int required_height,
+                          r2i::ImageFormat::Id required_format) override {
+    r2i::RuntimeError error;
+    int width;
+    int height;
+
+    this->required_width = required_width;
```

Comment:
I think you should verify if the requested dimensions for the out_frame are the supported by the proprocessing module, using the GetAvailableDataSizes method.

Reply:
Solved [here](https://github.com/RidgeRun/r2inference/commit/3289a0b9df6af6b18a3e4b7fabe3dc7d7f336efc)